runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -50,15 +58,37 @@ public IStreamView get(UUID stream) {
         return this.get(stream, StreamOptions.DEFAULT);
     }
 
+    public IStreamView getUnsafe(UUID stream) {
```

Comment:
Can we add a comment on how getUnsafe differentiates from get.. maybe implications/risks of using one versus the other or why we use this for streams consumed by the VLO.

Reply:
done.